lib/metadata.sh

```diff
@@ -4,21 +4,32 @@
 BUILD_DATA_FILE=""
 PREVIOUS_BUILD_DATA_FILE=""
 
-meta_create() {
+# Must be called before you can use any other methods
+meta_init() {
```

Comment:
The naming and usage here is still probably a little confusing, but we can bikeshed it when I try to upstream it to https://github.com/heroku/buildpack-stdlib